test/test_interface_ixxat.py

```diff
@@ -0,0 +1,68 @@
+"""
+Unittest for ixxat interface.
+
+Copyright (C) 2019 Marcel Kanter <marcel.kanter@googlemail.com>
+"""
+
+import unittest
+import can
+
+from can import CanError, CanBackEndError, CanInitializationError, CanOperationError
+
+class SoftwareTestCase(unittest.TestCase):
+    """
+    Test cases that test the software only and do not rely on an existing/connected hardware.

```

Comment:
Actually they rely on the ixxat driver installed on the maschine. Currently I'll skip the tests, if they can't be loaded (ImportError).

Still looking for a good and simple way for testing as much as possible without any driver.

Reply:
You could have a look at test_systec.py, which uses `unittest.mock`.

Reply:
I had a look onto the web-page of ixxat and did not find any linux driver. Maybe I'll find some time to call them this week. Since we use their adapters a lot at work, I'm very interested in a linux driver.

Reply:
They have linux drivers for x86 and AMD64 on their website.
I contacted HMS (Ixxat is a part of them) and got an beta driver for ARM64, I'm going to test them on the weekend on my ODROID.
Then we may get linux working support too.